src/cloud-api-adaptor/ibmcloud-powervs/image/copy-files.sh

```diff
@@ -6,27 +6,44 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 PODVM_DIR=${REPO_ROOT}/podvm
 
 sudo mkdir -p /etc/containers
-sudo cp /tmp/files/etc/aa-offline_fs_kbc-keys.json /etc/aa-offline_fs_kbc-keys.json
```

Comment:
Was this path mentioned somewhere in a docs? It changed but I don't see any change to where they were created (nor in the documentation). It'd be probably nice to mention it somewhere.